test/blackbox-tests/test-cases/variables-for-artifacts/run.t

```diff
@@ -32,10 +32,10 @@ prefixed and unprefixed modules are built.
 Command line version.
 
   $ ./sdune build --verbose %{cmo:a} %{cmo:b} %{cmo:c} 2>&1 | grep -A100 'Actual targets'
-  # Actual targets:
-  # - _build/default/.a1.objs/byte/a1__A.cmo
-  # - _build/default/.b.eobjs/byte/dune__exe__B.cmo
-  # - _build/default/.c1.objs/byte/c.cmo
```

Comment:
Were these hashes dropped on purpose?

Reply:
Kind of. To add them we have to:
- print to a string
- split the lines
- prefix the lines with `"# "`

So if did that for the console output, we'd have to:
- go from pp to a string
- split the lines
- prefix the lines
- package the result into a Pp again

which feels somewhat wrong :/

What I'd really like is a way to have a combinator:

```ocaml
val Pp.prefix_lines : 'a Pp.t -> prefix:string -> 'a Pp.t
```

I tried implementing it in the past but didn't managed to.

My thinking is that the `#` prefix for the terminal is not that useful so it's Ok to drop it. WDYT?

Reply:
Yeah, that seems a little too much effort to maintain some hashes in the output.

Perhaps `prefix_lines` is best implemented as a tag? The handler could be implemented in exactly the hacky way you described.

> which feels somewhat wrong :/

It doesn't feel wrong to me. The insertion of new lines is something that is decided at rendering time. For example, it would differ based on the wrapping margin that is chosen. So it seems like that's when this primitive should be interpreted.